sdk/core/Azure.Core/src/Shared/DiagnosticScopeFactory.cs

```diff
@@ -41,7 +41,7 @@ public DiagnosticScopeFactory(string clientNamespace, string? resourceProviderNa
 
         public bool IsActivityEnabled { get; }
 
-        public DiagnosticScope CreateScope(string name, DiagnosticScope.ActivityKind kind = DiagnosticScope.ActivityKind.Client)
+        public DiagnosticScope CreateScope(string name, DiagnosticScope.ActivityKind kind = DiagnosticScope.ActivityKind.Internal)
```

Comment:
We probably need to validate that all non-HTTP services are specifying a value for Kind already.

Reply:
I did a quick scan of SB and EH and that appears to be the case.
@jsquire can you double check my work?

Reply:
I double-checked and I believe SB and EH are good.